powerauth-restful-security-javaee/src/main/java/io/getlime/security/powerauth/rest/api/jaxrs/service/v3/ActivationService.java

```diff
@@ -294,7 +294,7 @@ public ActivationStatusResponse getActivationStatus(ActivationStatusRequest requ
             }
             return response;
         } catch (Exception ex) {
-            logger.warn("PowerAuth activation status check failed", ex);
+            logger.warn("PowerAuth activation status check failed, error: {}", ex.getMessage());
```

Comment:
Let's pass `ex` as a parameter to the new exception to keep the cause somewhere.